tf-modules/cumulus/variables.tf

```diff
@@ -188,6 +188,51 @@ variable "elasticsearch_alarms" {
   default = []
 }
 
+variable "ems_datasource" {
```

Comment:
There should be a description on all of these variables

Reply:
3377d11c5e